blocks/blog-home/blog-home.js

```diff
@@ -304,11 +288,8 @@ async function createFilters(categories, topics, audiences) {
   }
 
   // Topic filters
-  const topicsElement = document.createElement('div');
-  topicsElement.classList.add('topics');
-  topicsElement.setAttribute('aria-expanded', 'true');
-  const topicLabel = document.createElement('span');
-  topicLabel.classList.add('list-title');
+  const topicsElement = createTag('div', { class: 'topics', role: 'button', 'aria-expanded': 'true' });
+  const topicLabel = createTag('span', { class: 'list-title' });
```

Comment:
Nitpicking: This might be a difference in formatting preferences, but I'd like to format code like this (see below). To me, it's more concise and readable. Perhaps, we can configure JS linting to keep with this format. I'm open to suggestions.

``` js
const topicsElement = createTag('div', {
  class: 'topics',
  role: 'button',
  'aria-expanded': 'true'
});

const topicLabel = createTag('span', {
  class: 'list-title'
});
```